src/test/java/org/openelisglobal/patient/PatientContactServiceTest.java

```diff
@@ -0,0 +1,239 @@
+package org.openelisglobal.patient;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.sql.DataSource;
+import lombok.NonNull;
+import org.junit.Before;
+import org.junit.Test;
+import org.openelisglobal.BaseWebContextSensitiveTest;
+import org.openelisglobal.common.util.ConfigurationProperties;
+import org.openelisglobal.patient.service.PatientContactService;
+import org.openelisglobal.patient.valueholder.PatientContact;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+public class PatientContactServiceTest extends BaseWebContextSensitiveTest {
+
+    @Autowired
+    private PatientContactService patientContactService;
+
+    private JdbcTemplate jdbcTemplate;
+    private List<PatientContact> patientContacts;
+    private Map<String, Object> propertyValues;
+    private List<String> orderProperties;
+    private static int NUMBER_OF_PAGES = 0;
+
+    @Autowired
+    public void setDataSource(@NonNull DataSource dataSource) {
```

Comment:
Is this necessary for writing tests? Have you taken a look at other test classes and tried to see how the tests are written?